### PR TITLE
Feature: Add hollo world

### PR DESCRIPTION
**Pull request by yeshihai**
Feature: Add hollo world

### What?
brief: Add hollo world for users
author: Chunyang

### What changes are included in this PR?
添加了main.cpp实现hollo world，并在result图片中展示输出结果

### Are these changes tested?
已测试

### CODE REVIEW

**Comment by Light-City** [comment]
pr title style is worng.

**Comment by Light-City** [comment]
dir use lowercase underscase style ,scuch as: Test_Team -> test_team

File: stl/test_justwe/chunyang/src/main.cpp
Comment (martinx) [inline stl/test_justwe/chunyang/src/main.cpp:1]: Google Code Style:

```cpp
#include<iostream>
```

  => 1 space is needed

```cpp
#include <iostream>
```


```
int main() {
    // Hollow world
    std::cout << "Hollow world!" << std::endl;
}
```

=> 2 spaces

```cpp
int main() {
  // Hollow world
  std::cout << "Hollow world!" << std::endl;
}
```

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**